libs/components/toast/src/lib/modules/toast/toast.component.scss

```diff
@@ -106,47 +228,24 @@ sky-toast {
 
 .sky-theme-modern {
   sky-toast {
-    .sky-toast {
```

Comment:
I moved many of these styles up into the main declaration and used overrides when needed to clean up the code.